config/labels/analyzers/gcc.json

```diff
@@ -0,0 +1,285 @@
+      "profile:extreme",
+      "profile:security",
+      "profile:sensitive",
+      "severity:LOW"
+    ],
+    "gcc-fd-type-mismatch": [
+      "doc_url:https://gcc.gnu.org/onlinedocs/gcc/Static-Analyzer-Options.html#index-Wanalyzer-fd-type-mismatch",
+      "profile:extreme",
+      "profile:security",
+      "profile:sensitive",
+      "severity:LOW"
+    ],
+    "gcc-fd-use-after-close": [
+      "doc_url:https://gcc.gnu.org/onlinedocs/gcc/Static-Analyzer-Options.html#index-Wanalyzer-fd-use-after-close",
+      "profile:extreme",
+      "profile:security",
+      "profile:sensitive",
+      "severity:LOW"
+    ],
+    "gcc-fd-use-without-check": [
+      "doc_url:https://gcc.gnu.org/onlinedocs/gcc/Static-Analyzer-Options.html#index-Wanalyzer-fd-use-without-check",
+      "profile:extreme",
+      "profile:security",
+      "profile:sensitive",
+      "severity:LOW"
+    ],
+    "gcc-file-leak": [
+      "doc_url:https://gcc.gnu.org/onlinedocs/gcc/Static-Analyzer-Options.html#index-Wanalyzer-file-leak",
+      "profile:extreme",
+      "profile:security",
+      "profile:sensitive",
+      "severity:LOW"
+    ],
+    "gcc-free-of-non-heap": [
+      "doc_url:https://gcc.gnu.org/onlinedocs/gcc/Static-Analyzer-Options.html#index-Wanalyzer-free-of-non-heap",
+      "profile:extreme",
+      "profile:security",
+      "profile:sensitive",
+      "severity:LOW"
+    ],
+    "gcc-imprecise-fp-arithmetic": [
+      "doc_url:https://gcc.gnu.org/onlinedocs/gcc/Static-Analyzer-Options.html#index-Wanalyzer-imprecise-fp-arithmetic",
+      "profile:extreme",
+      "profile:security",
+      "profile:sensitive",
+      "severity:LOW"
+    ],
+    "gcc-infinite-recursion": [
+      "doc_url:https://gcc.gnu.org/onlinedocs/gcc/Static-Analyzer-Options.html#index-Wanalyzer-infinite-recursion",
+      "profile:extreme",
+      "profile:security",
+      "profile:sensitive",
+      "severity:LOW"
+    ],
+    "gcc-jump-through-null": [
+      "doc_url:https://gcc.gnu.org/onlinedocs/gcc/Static-Analyzer-Options.html#index-Wanalyzer-jump-through-null",
+      "profile:extreme",
+      "profile:security",
+      "profile:sensitive",
+      "severity:LOW"
+    ],
+    "gcc-malloc-leak": [
+      "doc_url:https://gcc.gnu.org/onlinedocs/gcc/Static-Analyzer-Options.html#index-Wanalyzer-malloc-leak",
+      "profile:extreme",
+      "profile:security",
+      "profile:sensitive",
+      "severity:LOW"
+    ],
+    "gcc-mismatching-deallocation": [
+      "doc_url:https://gcc.gnu.org/onlinedocs/gcc/Static-Analyzer-Options.html#index-Wanalyzer-mismatching-deallocation",
+      "profile:extreme",
+      "profile:security",
+      "profile:sensitive",
+      "severity:LOW"
+    ],
+    "gcc-null-argument": [
+      "doc_url:https://gcc.gnu.org/onlinedocs/gcc/Static-Analyzer-Options.html#index-Wanalyzer-null-argument",
+      "profile:extreme",
+      "profile:security",
+      "profile:sensitive",
+      "severity:LOW"
```

Comment:
obviously these warnings are not low level for your info type of warnings, but high level warnings. we will need to categorize them.

Reply:
Thats an admirable goal, but I'd strongly argue for postponing it after rc1.